app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/agents/AgentAccessController.scala

```diff
@@ -193,25 +194,28 @@ class AgentAccessController @Inject()(
     withVerifierMatchingDetails {
       case (agentSupplyingClientDetails, verifierMatchingDetails, _) =>
         if (verifierMatchingDetails.correctVerifierSupplied)
-          updateSession(sessionStore, request)(_.copy(journeyStatus = Some(
-            Subscribed(verifierMatchingDetails.clientDetails, agentSupplyingClientDetails.agentGGCredId)
-          ))).map{
+          updateSession(sessionStore, request)(
+            _.copy(
+              journeyStatus = Some(
+                Subscribed(verifierMatchingDetails.clientDetails, agentSupplyingClientDetails.agentGGCredId)
+              )
+            )
+          ).map {
             case Left(e) =>
-            logger.warn("Could not update session", e)
-            errorHandler.errorResult(request.userType)
+              logger.warn("Could not update session", e)
+              errorHandler.errorResult(request.userType)
```

Comment:
`errorHandler.errorResult(request.userType)` -> `errorHandler.errorResult()` as per #157

Reply:
I've changed that now